code/js/rcalc-constant-number-utility.js

```diff
@@ -0,0 +1,25 @@
+// size small in px
+const FONT_SIZE_SMALL_20_PX = 20;
+const FONT_SIZE_SMALL_8_PX = 8;
+const WIDTH_SMALL_30_PX = 30;
+const WIDTH_SMALL_93_PX = 93;
+const WIDTH_SMALL_125_PX = 125;
+const HEIGHT_SMALL_30_PX = 30;
+const HEIGHT_SMALL_25_PX = 25;
+// size default in px
+const FONT_SIZE_DEFAULT_40_PX = 40;
+const FONT_SIZE_DEFAULT_17_PX = 17;
+const WIDTH_DEFAULT_60_PX = 60;
+const WIDTH_DEFAULT_186_PX = 186;
+const WIDTH_DEFAULT_250_PX = 250;
+const HEIGHT_DEFAULT_60_PX = 60;
+const HEIGHT_DEFAULT_55_PX = 55;
+// size large in px
+const FONT_SIZE_LARGE_70_PX = 70;
+const FONT_SIZE_LARGE_30_PX = 30;
+const WIDTH_LARGE_100_PX = 100;
+const WIDTH_LARGE_305_PX = 305;
+const WIDTH_LARGE_410_PX = 410;
+const HEIGHT_LARGE_100_PX = 100;
+// zero﻿
+const NUMBER_ZERO = 0;
```

Comment:
Maybe only NUMBER_ZERO should be in file number utility. Others are related with styling. Try to do it better.

Reply:
all constants related to styling in 1 js file called rcalc-constant-html-css